ocp_resources/application_aware_applied_cluster_resource_quota.py

```diff
@@ -0,0 +1,12 @@
+# API reference: https://github.com/kubevirt/application-aware-quota/blob/main/README.md
```

Comment:
please add a link to the API (that's where the types are defined)

Reply:
no any docs yet, even upstream

Reply:
https://github.com/kubevirt/application-aware-quota/tree/bec951a92036a52693130d367845128c52a99ecb/staging/src/kubevirt.io/application-aware-quota-api/pkg/apis/core/v1alpha1

Reply:
done